txircd/ircd.py

```diff
@@ -372,7 +374,111 @@ def reloadModule(self, moduleName):
 		else:
 			deferList.addCallback(lambda result: self.loadModule(moduleName))
 		return deferList
-	
+
+	def verifyConfig(self, config):
+		# Verify the configuration values that are used in the core first.
+
+		# IRCd
+		if not isinstance(config["bind_client"], list):
+			raise ConfigValidationError("bind_client", "value must be a list")
+		for bindDesc in config["bind_client"]:
+			if not isinstance(bindDesc, str):
```

Comment:
In py2, the appropriate way to check for a string is with `isinstance(stringToCheck, basestring)`.